types/big.js/tsconfig.json

```diff
@@ -14,7 +14,8 @@
         ],
         "types": [],
         "noEmit": true,
-        "forceConsistentCasingInFileNames": true
+        "forceConsistentCasingInFileNames": true,
+        "esModuleInterop": true
```

Comment:
Normally, we prefer not to use `esModuleInterop` in DT configs, because it can allow non-portable import statements (ones that will error for consumers unless they too set `esModuleInterop`) to appear in the types themselves. However, these definitions have no imports in the types, and the tests do a good job of checking different ways of importing the module, in ways that should require `esModuleInterop` to be set. Setting these here allowed me to keep the imports in the tests as-is, which ensures that the module-related changes are not breaking for users as long as they have `esModuleInterop` set.